### DISCUSSION TITLE
Protecting Personal Information on GitHub: Why Privacy Matters for Content Creators and Students

### DISCUSSION THREAD

**Discussion by F0XN0T** [community/community#51269] [GitHub Education]
The issue of privacy and identity protection is a significant concern for content creators and students who use online platforms for their work. The case of an individual who encountered an issue while using GitHub for a school project related to photo manipulation highlights the importance of protecting personal information and privacy online.

As a community of coders, developers, and creators, it is crucial to raise awareness about this issue. Online platforms like GitHub should prioritize the protection of personal information and ensure that users have the option to use pseudonyms or creator names for public-facing communication.

While it is understandable that some platforms may require legal names for official purposes, users should have the choice to keep this information confidential and not shared with others. This can be particularly important for individuals who value their privacy, have safety concerns, or are working on sensitive projects.

As a community, we can advocate for the protection of personal information and the right to use pseudonyms or creator names online. We can also encourage platforms like GitHub to review their policies and guidelines to ensure that they prioritize privacy and security.

Have you faced similar problems while using online platforms? What are your thoughts on this issue, and how can we, as a community, raise awareness and advocate for the protection of personal information and privacy online? Let's start a discussion and work together to address this crucial issue.

**Comment by Z2Flow** [comment]
A new article in the Intercept casts a less-than-flattering light on Github's recent actions:

Check out: "No Subpoena Required?"

https://theintercept.com/2023/04/08/twitter-dmca-github-musk/
